ui/app/pages/swaps/select-quote-popover/sort-list/sort-list.js

```diff
@@ -71,94 +71,80 @@ export default function SortList ({
           className="select-quote-popover__column-header select-quote-popover__receiving"
           onClick={() => onColumnHeaderClick('destinationTokenValue')}
         >
-          <div className="select-quote-popover__receiving-header">
-            <span className="select-quote-popover__receiving-symbol">{swapToSymbol}</span>
-            <div className="select-quote-popover__receiving-label">
-              <span>{t('swapReceiving')}</span>
-              <InfoTooltip
-                position="bottom"
-                contentText={t('swapReceivingInfoTooltip')}
-              />
-              <ToggleArrows />
-            </div>
-          </div>
-        </div>
-        <div
-          className="select-quote-popover__column-header select-quote-popover__network-fees"
-          onClick={() => onColumnHeaderClick('rawNetworkFees')}
-        >
-          <div className="select-quote-popover__network-fees-header">
-            <span>{t('swapEstimatedNetworkFees')}</span>
+          <span className="select-quote-popover__receiving-symbol">{swapToSymbol}</span>
+          <div className="select-quote-popover__receiving-label">
+            <span>{t('swapReceiving')}</span>
             <InfoTooltip
               position="bottom"
-              contentText={t('swapEstimatedNetworkFeesInfo')}
+              contentText={t('swapReceivingInfoTooltip')}
             />
             <ToggleArrows />
           </div>
         </div>
+        <div
+          className="select-quote-popover__column-header select-quote-popover__network-fees select-quote-popover__network-fees-header"
+          onClick={() => onColumnHeaderClick('rawNetworkFees')}
+        >
+          <span>{t('swapEstimatedNetworkFees')}</span>
+          <InfoTooltip
+            position="bottom"
+            contentText={t('swapEstimatedNetworkFeesInfo')}
+          />
+          <ToggleArrows />
+        </div>
         <div
           className="select-quote-popover__column-header select-quote-popover__quote-source"
           onClick={() => onColumnHeaderClick('liquiditySource')}
         >
-          <>
```

Comment:
This was a particularly strange use of `<>` - the whole point of that construct is to pass a group of elements as a single element, in a case where a single element is expected (e.g. the top-level thing returned by `render`). But in this case, multiple elements are just fine :shrug: